### ISSUE TITLE
Scala 2.13: do not use deprecated API for collection conversion with Specific generation

### ISSUE THREAD

**Issue by fsarradin** [julianpeeters/avrohugger#135]
Hi,

I'm using sbt-avrohugger to generate case classes from Avro IDL. I've set the generation to avroScalaGenerateSpecific. But I get some problem as avrohugger is using deprecated conversion functions between Java and Scala collections.

Here is an example:

```avro
record ProductLifeCycle {
  string warehouseId;
  string productId;
  map<string> checkpoints;
}
```

With Scala 2.13 and by settings `-Xfatal-warning` compilation option, I get:

```
[error] /xxx/target/scala-2.13/src_managed/main/compiled_avro/xxx/data/meta/ProductLifeCycle.scala:39:30: object JavaConverters in package collection is deprecated (since 2.13.0): Use `scala.jdk.CollectionConverters` instead
[error]             scala.collection.JavaConverters.mapAsScalaMapConverter(map).asScala.toMap map { kvp =>
```

This seems to come from the class `avrohugger.format.specific.converters.ScalaConverter`.

**Comment by fsarradin** [comment]
Linked with https://github.com/julianpeeters/sbt-avrohugger/issues/78

**Comment by julianpeeters** [comment]
Fixed in avrohugger 1.0.0-RC25 and sbt-avrohugger 2.0.0-RC25.
